test/fixtures/indices.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  finalized_at: 2016-01-13 16:11:26
+  category_id: :one
```

Comment:
Shouldn't this be category: :one?  _id would expect an integer. (same for below on line 12)

Also, this field _might_ need to be renamed as above.